### PR TITLE
feat: Add actions menu redefinition

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
#### Screenshot or Gif

https://user-images.githubusercontent.com/20288327/125938828-a2fa41b6-e8de-4f74-b1dc-4657816e5fbd.mp4

#### Other relevant information
* Your OS: Linux Mint 19.1 Cinnamon x64. 
* Browser: Mozilla Firefox 88.0.1.
* Node.js version: 14.17.0.
* NPM version: 7.19.0.
* adempiere-vue version: 4.3.1.

### CODE REVIEW

File: src/components/ADempiere/ActionsMenu/MenuActions.vue [hunk 1]
```diff
@@ -0,0 +1,735 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
```
Comment (yamelsenih) [inline src/components/ADempiere/ActionsMenu/MenuActions.vue]: Rename to ActionMenu/Action

File: src/components/ADempiere/ActionsMenu/MenuReferences.vue [hunk 2]
```diff
@@ -0,0 +1,212 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+ Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
```
Comment (yamelsenih) [inline src/components/ADempiere/ActionsMenu/MenuReferences.vue]: Rename to ActionMenu/References

File: src/components/ADempiere/ActionsMenu/MenuRelations.vue [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
```
Comment (yamelsenih) [inline src/components/ADempiere/ActionsMenu/MenuRelations.vue]: Rename to ActionMenu/Relations

File: src/components/ADempiere/ActionsMenu/index.vue [hunk 4]
```diff
@@ -0,0 +1,120 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+ Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
```
Comment (yamelsenih) [inline src/components/ADempiere/ActionsMenu/index.vue]: Rename to ActionMenu/index.vue

File: src/lang/ADempiere/en/actionsMenu.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+const actionsMenu = {
+  refreshRecords: 'Refresh Records',
```
Comment (yamelsenih) [inline src/lang/ADempiere/en/actionsMenu.js]: Rename to actionMenu

File: src/lang/ADempiere/es/actionsMenu.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+
+const actionsMenu = {
```
Comment (yamelsenih) [inline src/lang/ADempiere/es/actionsMenu.js]: Rename to actionMenu

File: src/utils/ADempiere/constants/actionsMenu.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
+// ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+// Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
```
Comment (yamelsenih) [inline src/utils/ADempiere/constants/actionsMenu.js]: Delete it, will be implemented with own window

File: src/views/ADempiere/Test/MultiTabWindow/index.vue [hunk 8]
```diff
@@ -75,6 +76,40 @@ export default defineComponent({
       seekTab: function(eventInfo) {
         console.log('seekTab: ', eventInfo)
         return new Promise()
+      },
+
+      generateActionsMenu: (currentTab) => {
```
Comment (yamelsenih) [inline src/views/ADempiere/Test/MultiTabWindow/index.vue]: Rename to loadActionMenu

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/ActionsMenu/MenuActions.vue (hunk 1 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/ActionsMenu/MenuActions.vue]: Renamed

File: src/components/ADempiere/ActionsMenu/MenuRelations.vue (hunk 3 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/ActionsMenu/MenuRelations.vue]: Renamed

File: src/components/ADempiere/ActionsMenu/MenuReferences.vue (hunk 2 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/ActionsMenu/MenuReferences.vue]: Renamed

File: src/components/ADempiere/ActionsMenu/index.vue (hunk 4 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/ActionsMenu/index.vue]: Renamed folder

File: src/views/ADempiere/Test/MultiTabWindow/index.vue (hunk 8 above)
Comment (EdwinBetanc0urt) [reply src/views/ADempiere/Test/MultiTabWindow/index.vue]: method rename

File: src/lang/ADempiere/es/actionsMenu.js (hunk 6 above)
Comment (EdwinBetanc0urt) [reply src/lang/ADempiere/es/actionsMenu.js]: change lang

File: src/lang/ADempiere/en/actionsMenu.js (hunk 5 above)
Comment (EdwinBetanc0urt) [reply src/lang/ADempiere/en/actionsMenu.js]: Renamed

File: src/utils/ADempiere/constants/actionsMenu.js (hunk 7 above)
Comment (EdwinBetanc0urt) [reply src/utils/ADempiere/constants/actionsMenu.js]: I would apply the similar operation of 

https://github.com/EdwinBetanc0urt/adempiere-vue/blob/develop/src/components/ADempiere/Field/FieldOptions/fieldOptionsList.js

File: src/utils/ADempiere/constants/actionsMenuList.js [hunk 9]
```diff
@@ -0,0 +1,63 @@
+// ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+// Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
```
Comment (yamelsenih) [inline src/utils/ADempiere/constants/actionsMenuList.js:2]: Rename to actionMenuList

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/utils/ADempiere/constants/actionsMenuList.js (hunk 9 above)
Comment (yamelsenih) [reply src/utils/ADempiere/constants/actionsMenuList.js:2]: Remove instead

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Pull request merged**